spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/prometheus/PrometheusSimpleclientMetricsExportAutoConfiguration.java

```diff
@@ -69,14 +67,14 @@
 		before = { CompositeMeterRegistryAutoConfiguration.class, SimpleMetricsExportAutoConfiguration.class },
 		after = { MetricsAutoConfiguration.class, PrometheusMetricsExportAutoConfiguration.class })
 @ConditionalOnBean(Clock.class)
-@ConditionalOnClass(PrometheusMeterRegistry.class)
+@ConditionalOnClass(io.micrometer.prometheus.PrometheusMeterRegistry.class)
```

Comment:
What's the reason for that change?

Reply:
@mhalbritter I just wanted to align with the fully qualified return type of the `simpleclientPrometheusMeterRegistry()` below and remove the import statement for the `PrometheusMeterRegistry` that has been deprecated.